site_root/input/resources/search/SearchParameter-species.json

```diff
@@ -7,11 +7,14 @@
   "status": "active",
   "experimental": true,
   "publisher": "Kids First DRC",
-  "description": "Searches participants by species",
+  "description": "Searches Patients by species.",
```

Comment:
Hmm, I'm not sure about this one ... Should `Patients` -> `kfdrc-patients` because Patients cannot be searched by species without our profile?

Reply:
Updated and validated!